Add more videos and modify descriptions for current ones (#721)
wpilibsuite/frc-docs: source/docs/hardware/hardware-tutorials/sensors.rst

Sensing and Sensors
===================

> Without sensors and sensing robots are really radio controlled vehicles. Sensors allow the robots to understand the internal operation of the robots mechanical systems as well as the ability to interact with the environment around the robot. In these videos WPI Professor Craig Putnam describes a number of classes of sensors, how they are used, and tries to help you decide what sensors are best for your applications.

Reviewer: ```suggestion
Without sensors and sensing robots are really radio controlled vehicles. Sensors allow the robots to understand the internal operation of the robots mechanical systems as well as the ability to interact with the environment around the robot. In these videos, WPI Professor Craig Putnam describes a number of classes of sensors, how they are used, and provides guidance on what sensors are best for your applications.
```
- Comma after introductory phrase
- "Tries" is a weak word so I suggest we replace it.
